scripts/src/main/resources/scripts/command/python

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help version pip pip3 version -v --version"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+
+function doSetup() {
+  if [ ! -d "${PYTHON_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    # Get leatest release
+    if [ -z "${PYTHON_VERSION}" ]
+    then
+      doDebug "Python Home: ${PYTHON_HOME}"
+      doEcho "Getting latest release..."
+      # curl returns content of page, grep returns only version numbers, sort is sorting by version numbers and removing duplicates
+      # tail and head are returning the penultimate line, since it contains the latest stable version
+      PYTHON_VERSION=$(curl -L https://www.python.org/ftp/python 2>&1 | grep -o '3\.[0-9]*\.[0-9]*' | sort -u -V | tail -n2 | head -n1)
```

Comment:
While this seems to work, it is IMHO a "hack" and might break in the future.
If there is no "service" (URL) providing this more reliable, we usually simply hard-code the latest version as default in our commandlet. Examples:
https://github.com/devonfw/ide/blob/4a730466e00a5b6b632da2fa6f19a258707fee5c/scripts/src/main/resources/scripts/command/java#L27
https://github.com/devonfw/ide/blob/4a730466e00a5b6b632da2fa6f19a258707fee5c/scripts/src/main/resources/scripts/command/mvn#L18
https://github.com/devonfw/ide/blob/4a730466e00a5b6b632da2fa6f19a258707fee5c/scripts/src/main/resources/scripts/command/node#L21

Reply:
We will later address this with #826 so I set to resolved.